app/models/journal_manager.rb

```diff
@@ -136,7 +136,7 @@ def self.recreate_initial_journal(type, journal, changed_data)
     if journal.data.nil?
       journal.data = create_journal_data journal.id, type, changed_data.except(:id)
     else
-      journal.changed_data = changed_data
+      journal.details = changed_data
```

Comment:
Not sure If you already had a look at the specs and the reason is immediately clear. I had to look why this line fails in all related specs, and it is due to https://github.com/NobodysNightmare/openproject/blob/feature/aggregated_diffs/app/models/journal.rb#L59-L66 not being changed correctly.